### ISSUE TITLE
quickpick checkbox state not announced correctly by screen readers

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#43132]
1. enable a screen reader
2. run the `Manage Trusted Extensions` command.
3. checkboxes are always read as "unchecked"
